docker-test.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Setting up Docker test environment..."
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)